javascript/javascript2/week1/Javascript2Project/functionality.js

```diff
@@ -47,23 +47,35 @@ function updateScoreboard() {
   document.getElementById("cards-grid").style.display = "none";
   document.getElementById("timer").style.display = "none";
 
+  setTimeout(() => {
```

Comment:
I assume these `setTimeout` are being used to synchronize the appearance of the score values with the labels at the end of the game. It would be better to handle both using CSS animations. This approach would make the transitions more fluid and allow you to manage the animations in one central location.